webapp/packages/core-blocks/src/MenuPanel/MenuTrigger.tsx

```diff
@@ -220,7 +220,7 @@ interface IMenuInnerTriggerProps extends Omit<React.ButtonHTMLAttributes<any>, '
 
 export const MenuInnerTrigger = observer<IMenuInnerTriggerProps, HTMLButtonElement>(
   forwardRef(function MenuInnerTrigger(props, ref) {
-    const { menuItem, onItemClose, ...rest } = props;
+    const { menuItem, onItemClose, onClick, ...rest } = props;
     const menu = useMenuState();
     const style = useS(MenuPanelItemAndTriggerStyles);
```

Comment:
please don't modify core-blocks/src/MenuPanel/* it's deprecated and should be rewritten to context menu